R/estimate_infections.R

```diff
@@ -131,6 +133,24 @@ estimate_infections <- function(reported_cases,
                                 weigh_delay_priors = TRUE,
                                 id = "estimate_infections",
                                 verbose = interactive()) {
+  # Validate inputs
+  check_reports_valid(reported_cases, for_estimate_secondary = FALSE)
+  assert_class(generation_time, "generation_time_opts", null.ok = TRUE)
```

Comment:
just flagging that I think this should be `null.ok = FALSE` and the default argument for `generation_time` should be `generation_time_opts()` as in `estimate_infections()`, but that is an issue outside this PR.

Reply:
I've fixed the `null.ok = FALSE` issue. For the `generation_time` default, I'm guessing you meant to comment under `regional_epinow()`?